rudder-core/src/main/scala/com/normation/rudder/batch/AutomaticReportsCleaner.scala

```diff
@@ -372,11 +374,19 @@ case class AutomaticReportsCleaning(
 
           case ActiveCleaner =>
             val now = DateTime.now
-            val target = now.minusDays(ttl)
-            val formattedDate = formatDate(target)
+            val reportsCommand = DeleteCommand.Reports(now.minusDays(reportsttl))
```

Comment:
You shoudl do the same thing for reports command than compliance command 

 if (reportsttl > 0)  { Some( command) } else None

Reply:
yep

Reply:
Ok so we changed the condtion to run the automatic cleaning so reports ttl is always > 0 , no need to check it here